pkg/infrastructure/gcp/clusterapi/dns.go

```diff
@@ -144,10 +150,113 @@ func createDNSRecords(ctx context.Context, client *gcpic.Client, ic *installconf
 	return nil
 }
 
+// createPSCRecords will create and publish the records for the private service connect managed zone.
+func createPSCRecords(ctx context.Context, ic *installconfig.InstallConfig, clusterID string) error {
+	ctx, cancel := context.WithTimeout(ctx, time.Minute*2) // 1 minute for each record below
+	defer cancel()
```

Comment:
curious: Is it safe to assume 1 minute? I think it's way more than enough, but will it be OK to just use the passed-in context without timeout?

Reply:
I am going to leave this one for now. Timing issues are tricky and if we aren't having issues then lets not adjust this one.